plugin/tmux_navigator.vim

```diff
@@ -109,6 +113,18 @@ function! s:TmuxAwareNavigate(direction)
       endtry
     endif
     let args = 'select-pane -t ' . shellescape($TMUX_PANE) . ' -' . tr(a:direction, 'phjkl', 'lLDUR')
+    " Prevent wrapping, if option is set, by checking if last pane in appropriate direction
+    if g:tmux_navigator_no_wrap == 1
+      if a:direction == 'h'
```

Comment:
This can be condensed to just two lines using dictionaries:
```
let positions = {'h': 'left', 'j': 'bottom', 'k': 'top', 'l': 'right'}
let args = 'if -F "#{pane_at_' . positions[a:direction] . '}" "" "' . args . '"'
```
the definition for dictionary can then be moved to some global constant